platform/libs.asm/nbproject/project.xml

```diff
@@ -29,8 +29,12 @@
                 <subpackages>org</subpackages>
             </public-packages>
             <class-path-extension>
-                <runtime-relative-path>asm-all-5.0.1.jar</runtime-relative-path>
-                <binary-origin>external/asm-all-5.0.1.jar</binary-origin>
+                <runtime-relative-path>asm-7.2.jar</runtime-relative-path>
+                <binary-origin>external/asm-7.2.jar</binary-origin>
+            </class-path-extension>
+            <class-path-extension>
+                <runtime-relative-path>asm-tree-7.2.jar</runtime-relative-path>
+                <binary-origin>external/asm-tree-7.2.jar</binary-origin>
             </class-path-extension>
 <!--            <class-path-extension>
```

Comment:
Could be this obsolete part removed ?

Reply:
I will remove that part

Reply:
Which bit is obsolete, do we not need both asm & asm-tree?

Reply:
I think sdedic refers to the commented code in that file. The `asm-tree.jar` library have `org.objectweb.asm.tree` that it is used in `Asm.java` and `NbJShellAgent.java` files.

Reply:
@pepness correct